### ISSUE TITLE
Missing NSString constructor when generating the bindings for healthkit

### ISSUE THREAD

**Issue by leighajarett** [dart-lang/ffigen#484]
I am using ffigen to generate the bindings for the HealthKit ObjC framework. 

Here is the pubspec.yaml:

```
ffigen:
  name: HealthKit
  description: "Create bindings for the HealthKit APIs"
  language: objc
  output: "lib/healthkit_bindings.dart"
  compiler-opts:
    - '-F/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk/System/Library/Frameworks'
  exclude-all-by-default: true
  objc-interfaces:
    include:
      - "HKHealthStore"
      - "HKQuery"
  headers:
    entry-points:
      - "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk/System/Library/Frameworks/HealthKit.framework/Headers/HKHealthStore.h"
      - "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk/System/Library/Frameworks/HealthKit.framework/Headers/HKQuery.h"
```

In my dart file I am creating the library and attempting to convert a dart string to an NSString:

```
final lib = hk.HealthKit(DynamicLibrary.process());
var localeString = hk.NSString(lib, 'en_US')
```

But I get the error: `The class 'hk.NSString' doesn't have an unnamed constructor.`

Let me know if you need any more info!

**Comment by liamappelbe** [comment]
Can you attach your generated healthkit_bindings.dart file?

**Comment by leighajarett** [comment]
Github won't let me attach it as a dart file for some reason, so I copied it as a text file. LMK if you would prefer that I email the dart file or something

[healthkit_bindings copy.txt](https://github.com/dart-lang/ffigen/files/10022774/healthkit_bindings.copy.txt)

**Comment by dcharkes** [comment]
> Github won't let me attach it as a dart file for some reason, so I copied it as a text file. LMK if you would prefer that I email the dart file or something

We usually use 'gist's instead of attaching files: https://gist.github.com/

**Comment by leighajarett** [comment]
Here you go: https://gist.github.com/leighajarett/1efd873a9ee870498ed4748bbb1cc0f4

**Comment by liamappelbe** [comment]
The ObjCInterface.isBuiltIn flag must be bugged for your particular SDK/compiler. I'll see if I can repro

**Comment by dcharkes** [comment]
Thanks @liamappelbe !

I've published v7.2.1 please let us know if you're issue is fixed @leighajarett ! 🚀
